x/ccv/child/keeper/keeper_test.go

```diff
@@ -372,8 +372,13 @@ func TestKeeperTestSuite(t *testing.T) {
 // are registred and triggered when a validator has downtime
 func (suite *KeeperTestSuite) TestValidatorDowntime() {
 	// initial setup
-	app := suite.childChain.App.(*app.App)
-	ctx := suite.childChain.GetContext()
+	suite.SetupCCVChannel()
+	app, ctx := suite.childChain.App.(*app.App), suite.ctx
+	channelID := suite.path.EndpointA.ChannelID
+
+	// add parent channel to childkeeper store
+	app.ChildKeeper.SetParentChannel(ctx, channelID)
+	app.IBCKeeper.ChannelKeeper.SetChannel(ctx, types.PortID, channelID, suite.path.EndpointB.GetChannel())
```

Comment:
This line doesn't make sense. Why are you doing this here?

Reply:
Agreed, but it works to not have my tests panicing with:
`test panicked: parent channel not set: channel not found`

I'm still digging  for the issue into the consumer suite. I would be glad to have your view on this.